rotkehlchen/chain/manager.py

```diff
@@ -1569,6 +1571,60 @@ def query_ethereum_balances(
         self.query_ethereum_tokens()
         self._add_protocol_balances()
 
+    @cache_response_timewise()
+    def query_ethereum_lp_balances(
+            self,  # pylint: disable=unused-argument
+            balances: Dict[str, Dict[Asset, Balance]],
+            # Kwargs here is so linters don't complain when the "magic" ignore_cache kwarg is given
+            **kwargs: Any,
+    ) -> Optional[AddressToUniswapV3LPBalances]:
+        """Queries all ethereum liquidity pool balances and adds them to the balances.
+        Returns the Uniswap V3 LP balances as they're needed for the NFTs balances.
+        May raise:
+        - RemoteError if any of the calls fail.
+        """
+        uniswap = self.get_module('uniswap')
+        uniswap_v3_balances = None
+        if uniswap is not None:
+            uniswap_v3_balances = uniswap.get_v3_balances(
+                addresses=self.queried_addresses_for_module('uniswap'),
+            )
+            uniswap_v2_balances = uniswap.get_balances(
+                addresses=self.queried_addresses_for_module('uniswap'),
+            )
+            if len(uniswap_v2_balances) != 0:
+                if str(Location.BLOCKCHAIN) not in balances:
+                    balances[str(Location.BLOCKCHAIN)] = {}
+                for uniswap_v2_lps in uniswap_v2_balances.values():
+                    for uni_lp in uniswap_v2_lps:
+                        balances[str(Location.BLOCKCHAIN)][Asset(f'_ceth_{uni_lp.address}')] = uni_lp.user_balance  # noqa: E501
```

Comment:
no no no.

Here and below.

 Don't initialize an asset like `Asset(f'_ceth_{uni_lp.address}'` that ... ever! Check the code to see how we do it please.

We do `EthereumToken(uni_lp.address)`